keps/sig-network/1860-kube-proxy-IP-node-binding/kep.yaml

```diff
@@ -0,0 +1,22 @@
+title: Make Kubernetes aware of the load balancer behaviour
+kep-number: 1860
+authors:
+  - "@Sh4d1"
+owning-sig: sig-network
+participating-sigs:
+  - sig-cloud-provider
+reviewers:
+  - "@thockin"
+  - "@cheftako"
+  - TBD
+approvers:
+  - "@thockin"
+  - TBD
```

Comment:
Please add me as one of the approvers (representing SIG Cloud Provider)

Reply:
SGTM